src/views/articles.views.js

```diff
@@ -21,19 +21,21 @@ function listArticles(req, articlesTopic = [], actions = []) {
 }
 
 function renderArticle(req, article, actions = [], user, textHeaderActions) {
-  let userName = blocks.markdown(user?.displayName);
-  let userImage = blocks.image(user?.profilePicture);
+  let userName = user?.displayName ?? user?.username;
```

Comment:
maybe add another fallback like this
```suggestion
  let userName = user?.displayName ?? user?.username ?? "Unknown";
```

Reply:
In case it's empty, we just don't show the "submitted by" view